drivers/syslog/note_driver.c

```diff
@@ -79,10 +107,29 @@ static const struct file_operations note_fops =
  * Private Functions
  ****************************************************************************/
 
+/****************************************************************************
+ * Name: note_open
+ ****************************************************************************/
+
+#ifdef NOTE_FEATURE_BUFFER_REWINDABLE
+static int note_open(FAR struct file *filep)
+{
+  if ((filep->f_oflags & O_ACCMODE) != O_WRONLY)
+    {
+      /* Reset the pointer to read the trace data */
+
+      sched_note_trace_stop();
```

Comment:
Different to the existing sched_note feature, in task trace support, the period to collect the note (trace) data and to dump the trace result are completely separated.
While dumping the trace result the trace recording is stopped because task activity itself to dump the result affects the trace data.